.github/workflows/pr_gh_runner.yaml

```diff
@@ -0,0 +1,23 @@
+name: PR GitHub Runner
```

Comment:
"Launch Terraform Plan" ?

Reply:
In my opinion they are too long names for workflow titles. I like to keep them short and consistent, easy to navigate through the GitHub UI which shows just a little part of them. And, instead of saying what a workflow does, I'd like to say when/for what it should be used (usually validation or deployment)

Reply:
I cannot understand what the workflow does or when it should be used based on its current title ('PR GitHub Runner'). The title is very cryptic.

Reply:
I agree with @gunzip. What do you think about `PR infra runner`?

Reply:
Validating code changes in `github-runner` workspace, whatever they are

What is done in the CI workflow should not affect naming imho, because a) it could change over time and b) the user (developer) should not care but the CI result (green/red flag).

Something like "Lunch Terraform Plan" introduces confusion about the scenario where the workflow is intended to be used (pre merge? post merge? nightly? at deployment time? from which branch?)

I could propose `PR - GitHub Runner TF Validation`